elyra/contents/handlers.py

```diff
@@ -74,3 +74,6 @@ def _get_absolute_path(self, path: str) -> str:
             path = path[1:]
         absolute_path = os.path.normpath(os.path.join(root_dir, path))
         return absolute_path
+
+    def write_error(self, status_code, **kwargs):
```

Comment:
Is something flagging these as necessary?  Seems like `write_error()` resolution would be satisfied by the mixin.

Reply:
CodeQL is showing warnings for this. seems they want you to explicitly define which method we use despite it always resolving from left to right in terms of inheritance. Guess its for clarity.